src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+  gap: 4px;
+}
```

Comment:
The 'gap' property is not fully supported in all browsers (e.g., Internet Explorer). It's better to use 'margin-right' for '.stars__star' and then override it for the last child.